### ISSUE TITLE
GetBalance API operation not available?

### ISSUE THREAD

**Issue by mpetyx** [paypal/PayPal-Python-SDK#206]
Hello,

I just wanted to ask if [this](https://developer.paypal.com/docs/classic/api/merchant/GetBalance_API_Operation_NVP/) operation is available within this SDK.
I couldn't find anything related to GetBalance here, and I am a little bit worried to be honest.

Thank you in advance, for your response.

Regards,
Michael.

**Comment by xiaoleih41** [comment]
GetBalance API is not available in REST API at this time.  Can you please share with me your use case, I can submit a feature request on this. In the meantime, you may have to use the NVP version instead.

**Comment by mpetyx** [comment]
Thanks @xiaoleih41!
Our case is as simple as that.
We have a platform that enables people to earn money.
All the earnings are saved in our balance, and whenever a user wants to retrieve it, we send them automatically through the API.
The problem is, that whenever our balance is empty it just fails silently.
Thus, we need to check that our balance is ok before we start a transaction, otherwise to manually top up.

**Comment by braebot** [comment]
This issue is now at https://github.com/paypal/PayPal-REST-API-Issues/issues/90. Follow this new issue to receive updates.
